geoportal/c2cgeoportal_geoportal/scripts/create_demo_theme.py

```diff
@@ -43,38 +45,38 @@ def main():
         description="Create and populate the database tables."
     )
     fill_arguments(parser)
-
     options = parser.parse_args()
 
-    get_app(options, parser)
-
-    from c2cgeoportal_commons.models import DBSession
-    from c2cgeoportal_commons.models.main import Interface, OGCServer, Theme, LayerGroup, LayerWMS
+    config_uri = get_config_uri(options)
+    loader = get_config_loader(config_uri)
+    loader.setup_logging()
+    settings = loader.get_wsgi_app_settings()
```

Comment:
Those 4 lines still be in all the scripts...

Reply:
Yes, codacy also complains about thoses lines.
From my point of view, each line relates to something really different, so it make sense for me to keep them.
- First line is specific to c2cgeoportal for the parameters from scripts packages.
- Second line is a getter coming from pyramid.
- Third line is an action.
- Fourth line might be different in another script (for example get the app instead of settings).
It would be wreid to setup logging in a `get_settings` method.
IMHO we should keep thoses lines.

Reply:
Yes, but it's relay not script specific, and finally in the script we just want to have the settings, and don't needs to know this complexity...